packages/twilio-run/__tests__/config/global.test.ts

```diff
@@ -145,4 +145,69 @@ describe('readSpecializedConfig', () => {
       env: '.env.stage',
     });
   });
+
+  test('account + region config override', () => {
+    __setTestConfig({
+      serviceSid: 'ZS11112222111122221111222211112222',
```

Comment:
the sid convention is a little confusing here, is this service sid supposed to match up with any of the ones below? or does only the service sids in `projects` matter for this test?